gluten-core/src/main/scala/org/apache/gluten/extension/columnar/enumerated/EnumeratedApplier.scala

```diff
@@ -56,10 +56,8 @@ class EnumeratedApplier(
     finalPlan
   }
 
-  private def apply0(rules: Seq[Rule[SparkPlan]], plan: SparkPlan): SparkPlan = {
-    val executor = new ColumnarRuleApplier.Executor("ras", rules)
-    executor.execute(plan)
-  }
+  private def apply0(rules: Seq[Rule[SparkPlan]], plan: SparkPlan): SparkPlan =
+    new ColumnarRuleApplier.ColumnarRuleExecutor("ras", rules).execute(plan)
```

Comment:
`ColumnarRuleApplier.ColumnarRuleExecutor` sounds a little bit too long. Let's move `ColumnarRuleExecutor` to an individual file, i.e., `ColumnarRuleExecutor.scala`? Thanks.